ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -194,7 +194,7 @@ void window_set_fullscreen(bool full) {
     window_set_maximized(false);
     tmpWidth = window_get_width();
     tmpHeight = window_get_height();
-    window_set_stayontop(true);
+    window_set_stayontop(false);
```

Comment:
It might be best to just not take a stance on this and delete this line. But in the interest of protecting end users, I suppose this is fine.